docs/en/setup/examples/grpc-load-balance-client/README.md

```diff
@@ -0,0 +1,84 @@
+# GRPC load balance client
+
+GRPC client support connect to multiple server address, and using round-robin to load-balance server before send each request.
```

Comment:
```suggestion
GRPC client support connect to multiple server address, and use `round-robin` policy for load-balance.
```
Does this base on per streaming or per message?

Reply:
Per streaming.